datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -867,6 +867,15 @@ def check(self, instance):
         # type: (InstanceType) -> None
         raise NotImplementedError
 
+    def cancel(self):
+        """
+        This method is called when the check in unscheduled by the agent. This
```

Comment:
```suggestion
    def cancel(self):
        # type: () -> str
        """
        This method is called when the check in unscheduled by the agent. This
```

Reply:
The method should not return anything. Should I use `type: () -> None` ?

Reply:
Yeah bad copy/paste on my end. `type: () -> None` is the correct thing to add